modules/hostdb.go

```diff
@@ -7,7 +7,7 @@ import (
 
 const (
 	// Denotes a host announcement in the Arbitrary Data section.
-	HostAnnouncementPrefix = "HostAnnouncement"
+	PrefixHostAnnouncement = "HostAnnouncement\n"
```

Comment:
Makes it easier to read in plaintext. Not sure how useful that will end up being but I figured if the prefix is going to be human readable it might as well be formatted.

It's conceivable that someone would be viewing raw data in a text editor.